src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -132,60 +133,64 @@ class Microsoft_Launcher : public PowertoyModuleIface {
    // Enable the powertoy
   virtual void enable()
   {
```

Comment:
The only change made on this file was the following:
Earlier it was

    enable() 
    {
      logic; 
      m_enabled=true;
    }

Now it is

    enable() 
    {
      if (UseNewSettings())
      {
        logic;
      }
      m_enabled = true;
    }